sdk/react/bottomOverlay.js

```diff
@@ -149,8 +149,8 @@ class BottomOverlay extends Component {
         testID={VIEW_NAMES.TIME_SEEK_BAR_THUMB}
         accessible={false}
         importantForAccessibility='no-hide-descendants'
-        accessibilityLabel={VIEW_NAMES.TIME_SEEK_BAR_THUMB}
-        style={[scrubberStyle, positionStyle, { width: scrubberSize, height: scrubberSize }]}>
+        accessibilityLabel=''
```

Comment:
I suppose we have to specify accessibilityLabel, so the left option seems more suitable, doesn't it?

Reply:
It is intended change, we need cut any possibility to voice that text but if we just remove whole accessibilityLabel property control become unfocusable. Actually it is some kind of "hack" because 
importantForAccessibility='no-hide-descendants' 
just don't work correctly of some of QA android device correctly